resources/views/components/progress-bar/index.blade.php

```diff
@@ -11,11 +11,11 @@ class="flex items-center space-x-[18px] text-xs"
     </span>
     @foreach (array_slice(config('rapidez.checkout_steps.' . config('rapidez.store_code')) ?? config('rapidez.checkout_steps.default'), 0, -1) as $stepTitle)
         <div
-            class="aspect-square w-3 cursor-pointer rounded"
+            class="aspect-square w-3 rounded"
             v-on:click="goToStep({{ $stepTitle == 'Credentials' ? 3 : $loop->index }})"
             :class="{
-                'bg-ct-accent': {{ $loop->index + 1 }} <= checkout.step,
-                'bg-ct-border': {{ $loop->index + 1 }} > checkout.step,
+                'bg-ct-accent cursor-pointer': {{ $loop->index + 1 }} <= checkout.step,
+                'bg-ct-border pointer-events-none': {{ $loop->index + 1 }} > checkout.step,
```

Comment:
maybe better to make it cursor-not-allowed?

Reply:
pointer-events-none prevents settings the cursor, I think this is a better solution since the clickable areas are clickable because of the cursor-pointer